src/pages/MoviesList/MovieApp.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable prefer-const */
+import React, { useEffect, useState } from "react";
+import MovieList from "components/MovieList/MovieList";
+import Footer from "components/Footer/Footer";
+import SidePanel from "components/SidePanel/SidePanel";
+import { Container } from "pages/MoviesList/styles";
+import axios from "axios";
+import Header from "components/Header/Header";
+/**
```

Comment:
Could you try to add an empty space after each block of code, this makes the code more organized and easier to differentiate separate blocks of code.